Makefile

```
@@ -151,6 +151,9 @@ firefox_install: firefox_deps bzip2 wget java
 firefox_deps:
 	sudo apt-get install -qqy --no-install-suggests $$(apt-cache depends firefox | grep Depends | sed "s/.*ends:\ //" | tr '\n' ' ')
 
+geckodriver: node-selenium-standalone
+	./webapp/node_modules/.bin/selenium-standalone install --singleDriverInstall=firefox
```

Hmm, I thought this was installed in the `post-install` script of `selenium-standalone`, no?

If we need to do this, we should:
```suggestion
	cd webapp; `npm bin`/selenium-standalone install --singleDriverInstall=firefox
```